tutorial/consuming_packages/cross_building_with_conan.rst

```diff
@@ -168,7 +168,7 @@ dependency built for *armv7hf* architecture and a **cmake/3.22.6** version that
 
 .. code-block:: bash
     
-    $ conan install . --build missing -pr:b=default -pr:h=./profiles/raspberry
+    $ conan install . --output-folder=build --build missing -pr:b=default -pr:h=./profiles/raspberry
```

Comment:
Not necessary, the recipe contains a ``cmake_layout(self)`` that automatically defines the "build" folder

Reply:
oh. Sorry. I have used conanfile.txt instead conanfile.py